packages/near-fast-auth-signer/src/components/CreateAccount/CreateAccount.tsx

```diff
@@ -110,14 +71,11 @@ const schema = yup.object().shape({
     .test(
       'is-account-available',
       async (username, context) => {
-        if (username) {
-          const isAvailable = await checkIsAccountAvailable(username);
-          if (!isAvailable) {
-            return context.createError({
-              message: `${username}.${network.fastAuth.accountIdSuffix} is taken, try something else.`,
-              path:    context.path
-            });
-          }
+        if (username && await window.fastAuthController.accountExist(username)) {
```

Comment:
I believe this part of logic has been re-written recently

Reply:
By what I checked, we only changed the e-mail validation logic. The username validation remains the same.